GPT/beta-commands/beta-cursorless.talon

```diff
@@ -14,3 +14,21 @@ app: vscode
     text_list = user.cursorless_get_text_list(cursorless_target)
     default_destination = user.cursorless_create_destination(cursorless_target)
     user.gpt_dynamic_request_cursorless(user.text, text_list, cursorless_destination or default_destination)
+
+model blend <user.cursorless_target> to <user.cursorless_target>:
+    target_text = user.cursorless_get_text_list(cursorless_target_1)
+    destination_text = user.cursorless_get_text(cursorless_target_2)
+    default_destination = user.cursorless_create_destination(cursorless_target_2)
+    result = user.model_blend_list(target_text, destination_text)
+    user.cursorless_insert(default_destination, result)
+
+model blend to <user.cursorless_target>:
+    target_text = edit.selected_text()
+    destination_text = user.cursorless_get_text(cursorless_target)
+    default_destination = user.cursorless_create_destination(cursorless_target)
+    result = user.model_blend(target_text, destination_text) * user.cursorless_insert(default_destination, result)
```

Comment:
What's going on with this line?

Reply:
Looks like a typo that got reformatted automatically. Just fixed it-thanks!